### DISCUSSION TITLE
Tailing cloudwatch logs

### DISCUSSION THREAD

**Discussion by mrichards01** [aws/aws-toolkit-jetbrains#3673] [Feature requests]
This is perhaps a usability question but I have found that tailing logs in the cloudwatch logs feature doesn't work so well for me. Often times I use the AWS toolkit within intellij to tail cloudwatch logs for different environments rather than use the console or CLI.

It can help to simplify this process as much as possible and reduce the number of steps and clicks to do this or understand why the 'tail logs' button is not working for me and whether others have similar issues with this. A brief description of my current work flow:

1)  I select the latest log stream available in the cloudwatch group
2) I click the native intellij 'tail logs' button to the left of the window. Unexpectedly, I find it doesn't render all of the available log lines and it doesn't auto-scroll to the bottom of all the available log lines.
3) I resort to scrolling all the way to the bottom of my logs.

First and foremost, I wanted to start a discussion. Happy to add more details and even contribute to a potential feature to fix this. I couldn't find matches to related discussions or features.

**Comment by lambda-science** [comment]
Ever find a fix ? I have the same issue !

**Comment by lambda-science** [comment]
Would be great if newer logs could be at top !
